projects/sw-offline-google-analytics/src/lib/replay-queued-requests.js

```diff
@@ -27,46 +26,48 @@ const idbHelper = new IDBHelper(constants.IDB.NAME, constants.IDB.VERSION,
  * Returns a promise that resolves when the replaying is complete.
  *
  * @private
- * @param {Object=} additionalParameters URL parameters, expressed as key/value
+ * @param {Object=} parameterOverrides URL parameters, expressed as key/value
  *                 pairs, to be added to replayed Google Analytics requests.
  *                 This can be used to, e.g., set a custom dimension indicating
  *                 that the request was replayed from the service worker.
  * @returns {Promise.<T>}
  */
-module.exports = additionalParameters => {
-  additionalParameters = additionalParameters || {};
-
+module.exports = ({parameterOverrides = {}, hitFilter} = {}) => {
   return idbHelper.getAllKeys().then(urls => {
     return Promise.all(urls.map(url => {
       return idbHelper.get(url).then(hitTime => {
         const queueTime = Date.now() - hitTime;
         const newUrl = new URL(url);
 
-        // URLSearchParams was added in Chrome 49.
-        // On the off chance we're on a browser that lacks support, we won't
-        // set additionParameters, but at least we'll set qt=.
-        if ('searchParams' in newUrl) {
-          additionalParameters.qt = queueTime;
-          // Call sort() on the keys so that there's a reliable order of calls
-          // to searchParams.set(). This isn't important in terms of
-          // functionality, but it will make testing easier, since the
-          // URL serialization depends on the order in which .set() is called.
-          Object.keys(additionalParameters).sort().forEach(parameter => {
-            newUrl.searchParams.set(parameter, additionalParameters[parameter]);
-          });
-        } else {
-          log('The browser does not support URLSearchParams, ' +
-            'so not setting additional parameters.');
-          newUrl.search += (newUrl.search ? '&' : '') + 'qt=' + queueTime;
-        }
+        // Do not attempt to replay hits that are too old.
+        if (queueTime > constants.STOP_RETRYING_AFTER) return;
```

Comment:
It doesn't enforce it because my way is better :)

But I'll update for consistency. I can also remove this particular condition if we want based on https://github.com/GoogleChrome/sw-helpers/pull/39#discussion_r74503628.